src/main/java/seedu/address/logic/commands/AddSellerCommand.java

```diff
@@ -26,14 +30,22 @@ public class AddSellerCommand extends Command {
             + PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
-            + PREFIX_ADDRESS + "ADDRESS "
+            + PREFIX_HOUSINGTYPE + "HOUSINGTYPE"
+            + PREFIX_STREET + "STREET "
+            + PREFIX_BLOCK + "BLOCK "
+            + PREFIX_LEVEL + "LEVEL "
+            + PREFIX_UNITNUMBER + "UNITNUMBER "
             + PREFIX_POSTALCODE + "POSTAL CODE "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
+            + PREFIX_HOUSINGTYPE + "HDB "
+            + PREFIX_STREET + "Clementi Ave 2 "
+            + PREFIX_BLOCK + "Block 311 "
```

Comment:
I think this part has to be changed, because the prefix_block + "Block 311 " should be "311 " instead, since the block can only take in numbers and one letter. I believe that it should be the changes should also be made to level and unit number.

But this one is quite minor since it is the UI, can be changed later.